scripts/install.sh

```diff
@@ -25,6 +25,7 @@ nixos-generate-config --root /mnt
 curl -f "$packer_http/vagrant.nix" > /mnt/etc/nixos/vagrant.nix
 curl -f "$packer_http/vagrant-hostname.nix" > /mnt/etc/nixos/vagrant-hostname.nix
 curl -f "$packer_http/vagrant-network.nix" > /mnt/etc/nixos/vagrant-network.nix
+curl -f "$packer_http/custom-configuration.nix" > /mnt/etc/nixos/custom-configuration.nix
```

Comment:
It would be good to only include the target guest config. I don't know how much hyperv support adds to the closure size?

Eg: `curl -f "$packer_http/guest/$packet_target.nix" > /mnt/etc/nixos/guest.nix` (`$packet_target` is probably wrong but you get the idea)

Reply:
I am surprised. I would believe `virtualisation.hypervGuest.enable = false;` would do nothing ? The idea was to remove this specific configuration out of configuration.nix

Reply:
I didn't look at the code close enough. Now I see the `sed` commands.

I still think that it would be best to have one file per virtualization technology as the "sed" clauses will be quite fragile.

Reply:
I introduced the mechanism in #37 to make it easier for you